activesupport/lib/active_support/testing/parallelization.rb

```diff
@@ -81,6 +83,8 @@ def start
               reporter = job[2]
               result   = Minitest.run_one_method(klass, method)
 
+              add_setup_exception(result, setup_exception) if setup_exception.present?
```

Comment:
Don’t think we need present? here.

Reply:
Good call, I don't believe we do.  Removed in ccc2ed63c7